src/transaction/transaction_group_complete_manager.hpp

```diff
@@ -47,6 +47,17 @@ namespace cubtx
 			| GROUP_COMPLETE_STARTED | GROUP_COMPLETED)
   };
 
+  //
+  // group completion is the common interface used to control group creation.
+  //
+  class group_completion
```

Comment:
It doesn't look a good fit of name and file. Generic group complete manager is not directly linked to stream/stream position.

It should belong to cubreplication namespace and it should be moved to replication/slave/apply related files. One quick action could be moving to slave GCM file, rename it as slave_group_complete_controller. Its name and place can be later reconsidered its with http://jira.cubrid.org/browse/CBRD-23218.

Reply:
This is my only remaining unresolved comment.

Reply:
Please clarify with Razvan your suggestion. Moving into slave GCM is not suitable since group_completion is accessed by log_consumer. Renaming again, seems to be an issue without end...

Reply:
Since for some of us looks important, finally, was moved n a new file without changing name.